azure-pipelines-woa.yml

```diff
@@ -63,15 +63,14 @@ steps:
     set npm_config_nodedir=%cd%\out\Default\gen\node_headers
     set npm_config_arch=arm64
     cd electron
-    node script/yarn test -- --enable-logging --verbose
+    FOR /L %%x IN (1,1,10) DO (node script/yarn test -- --enable-logging --verbose)
```

Comment:
is this for debugging purposes?

Reply:
LOL - yes - and why this PR is in draft

Reply:
I've run the tests locally on a WOA machine 10 times but I want to make sure its good to go through CI.